etc/docs/api.proto

```diff
@@ -0,0 +1,62 @@
+service ExecutorService{
+  rpc ExecuteCommand (ExecuteRequest) returns (ExecuteReply) {}
+  rpc StopExecution (ExecutionIdRequest) returns (StopCommandReply) {}
+  rpc ListExecutions (EmptyRequest) returns (ExecutionListReply) {}
+  rpc GetExecution (ExecutionIdRequest) returns (ExecutionInfoReply) {}
+  rpc GetOutStream(ExecutionIdRequest) returns (stream OutChunk) {}
+}
+
+message ExecuteRequest {
+  required string command = 1;
```

Comment:
nit: if you use proto3, `required` is no longer needed

Reply:
that's a valid point, i could also use google's Empty for the List request